Sources/Common/Service/HttpRequestParams.swift

```diff
@@ -1,13 +1,27 @@
 import Foundation
 
 public struct HttpRequestParams {
-    public let endpoint: HttpEndpoint
+    public let method: String
+    public let url: URL
```

Comment:
Refactored this object to be used for CIO API endpoints and for publicly accessible URLs when downloading files from a CDN.